catalog/datasets/stcn-drukkers.jsonld

```diff
@@ -0,0 +1,37 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "http://data.bibliotheken.nl/id/dataset/stcn/printers",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "STCN: drukkers"
```

Comment:
Good name? Do you agree with abbreviating to STCN?

Reply:
Maybe use "Short-Title Catalogue, Netherlands (STCN) - Drukkers"? 
This is of course an awkward mix of English and Dutch but here is no Dutch translation for STC.

Reply:
For the sake of consistency, let’s stick to colon `:` to separate levels (e.g. `GTAA: onderwerpen`).